src/main/kotlin/io/github/vootelerotov/renovator/Renovator.kt

```diff
@@ -81,48 +80,90 @@ class Renovator : CliktCommand() {
 
   private fun merge(pr: PullRequest, pullRequestClient: PullRequestClient, confirmedViaFlag: Boolean = false) {
     if (pr.merged() != false) {
-      echo("PR ${prDescription(pr)} is already merged")
+      echo("PR with ${prDescriptionShort(pr)} is already merged")
       return
     }
 
     if (pr.mergeable().getOrNull() != true) {
-      echo("PR ${prDescription(pr)} cannot be merged")
+      echo("PR with ${prDescriptionShort(pr)} cannot be merged")
       return
     }
 
-    val isConfirmed = confirmedViaFlag || confirm("Approve and merge PR ${prDescription(pr)}?")
-      ?: throw IllegalStateException("Can not get confirmation from stdin")
+    val isConfirmed = confirmedViaFlag || confirmMerge(pr)
 
     if (!isConfirmed) {
       return
     }
 
-    val comment = prompt("Enter comment to approve the PR with", default = defaultComment)
+    val comment = if (defaultComment.isNullOrBlank()) {
+      prompt("Enter comment to approve the PR with")
+    } else {
+      prompt(
+        "Press enter to approve the PR with the default comment '$defaultComment' or enter a different comment to approve PR with",
+        default = defaultComment,
+        showDefault = false
+      )
+    }
       ?: throw IllegalStateException("Can not get comment from stdin")
 
-    echo("Approving PR ${prDescription(pr)} with comment $comment")
+    echo("Approving PR ${pr.title()} with comment $comment")
 
     pullRequestClient.createReview(pr.number()!!, ImmutableReviewParameters.builder()
       .body(comment)
       .event(APPROVE_EVENT)
       .build()
     ).get()
-    echo("Merging PR ${prDescription(pr)} via re-base")
+    echo("Merging PR ${pr.title()} via re-base")
     pullRequestClient.merge(
       pr.number()!!,
       ImmutableMergeParameters.builder().mergeMethod(MergeMethod.rebase).sha(pr.head()?.sha()!!).build()
     ).get()
     echo("")
   }
 
+  private fun confirmMerge(pr: PullRequest): Boolean {
+    val input = prompt(
+      "Approve and merge following PR [y/N]? Type 'i' to get more information about the PR. ${prDescriptionLong(pr)}",
```

Comment:
Two ideas:
1) What about having the prDescripiton before the prompt? Usually, the promp is the last thing you do, here we would create distance between the options and when you have to type them in.
2) What about moving the information on what `i` means to `?` option? 

So:

```
"${prDescriptionLong(pr)} Approve and merge this PR [y/N/i/?]",
```

Can not be polite and use `?` in the sentence then though, or there are two `?` with different semantics too close.